### ISSUE TITLE
setting Fps to 10, Output Fps is still 30

### ISSUE THREAD

**Issue by VansonLeung** [shogo4405/lf.swift#48]
I am using master branch demo.
Then I added in the demo:

        rtmpStream.videoSettings = [
            "width" : 120,
            "height" : 160,
            "fps" : 10,
        ]

And then I play the output with rtmp://192.168.1.101:14000/hls/movie

Unfortunately the output Video is still 30 fps

**Comment by shogo4405** [comment]
fps not supported yet.

I changed api. see https://github.com/shogo4405/lf.swift/commit/9c6c9bb4a7610605388bd3873e6ff6080306ee93 debugging now.
```sh 
rtmpStream.captureSettings = [
    "fps" : 15,  // 29.97
]
```

**Comment by shogo4405** [comment]
9c6c9bb
oops. Wowza Error.
```txt
Zero sized function (client:1671870522:FME/3.0 (compatible; FMSc/1.0)): type:20 size:0:
```

**Comment by shogo4405** [comment]
released 0.3.1
